[2023-11-02T09:54:27.173-0500] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: economics_data_pipeline.pull_and_save_fred_data scheduled__2023-04-09T00:00:00+00:00 [queued]>
[2023-11-02T09:54:27.178-0500] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: economics_data_pipeline.pull_and_save_fred_data scheduled__2023-04-09T00:00:00+00:00 [queued]>
[2023-11-02T09:54:27.178-0500] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-11-02T09:54:27.186-0500] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): pull_and_save_fred_data> on 2023-04-09 00:00:00+00:00
[2023-11-02T09:54:27.207-0500] {standard_task_runner.py:57} INFO - Started process 96911 to run task
[2023-11-02T09:54:27.218-0500] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'economics_data_pipeline', 'pull_and_save_fred_data', 'scheduled__2023-04-09T00:00:00+00:00', '--job-id', '32', '--raw', '--subdir', 'DAGS_FOLDER/economics_data_pipe.py', '--cfg-path', '/var/folders/3l/sjtljk5j23ng0k_w12x41kh80000gn/T/tmpq2xegjva']
[2023-11-02T09:54:27.219-0500] {standard_task_runner.py:85} INFO - Job 32: Subtask pull_and_save_fred_data
[2023-11-02T09:54:27.256-0500] {task_command.py:416} INFO - Running <TaskInstance: economics_data_pipeline.pull_and_save_fred_data scheduled__2023-04-09T00:00:00+00:00 [running]> on host macbook-pro-207.local
[2023-11-02T09:54:27.296-0500] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='hermesadmin' AIRFLOW_CTX_DAG_ID='economics_data_pipeline' AIRFLOW_CTX_TASK_ID='pull_and_save_fred_data' AIRFLOW_CTX_EXECUTION_DATE='2023-04-09T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-04-09T00:00:00+00:00'
[2023-11-02T09:54:27.297-0500] {fred_utils.py:34} INFO - Fetching data from URL: https://api.stlouisfed.org/fred/series/observations?series_id=GDP&api_key=5c78bd237c041fcf6bebdae4f8e05905&file_type=json
[2023-11-02T10:06:05.944-0500] {local_task_job_runner.py:294} WARNING - State of this instance has been externally set to None. Terminating instance.
[2023-11-02T10:06:05.950-0500] {process_utils.py:131} INFO - Sending Signals.SIGTERM to group 96911. PIDs of all processes in the group: [96911]
[2023-11-02T10:06:05.950-0500] {process_utils.py:86} INFO - Sending the signal Signals.SIGTERM to group 96911
[2023-11-02T10:07:05.977-0500] {process_utils.py:149} WARNING - process psutil.Process(pid=96911, name='Python', status='running', started='09:54:27') did not respond to SIGTERM. Trying SIGKILL
[2023-11-02T10:07:05.981-0500] {process_utils.py:86} INFO - Sending the signal Signals.SIGKILL to group 96911
[2023-11-02T10:07:05.989-0500] {process_utils.py:79} INFO - Process psutil.Process(pid=96911, name='Python', status='terminated', exitcode=<Negsignal.SIGKILL: -9>, started='09:54:27') (96911) terminated with exit code Negsignal.SIGKILL
[2023-11-02T10:07:05.990-0500] {standard_task_runner.py:172} ERROR - Job 32 was killed before it finished (likely due to running out of memory)
[2023-11-02T10:07:13.497-0500] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: economics_data_pipeline.pull_and_save_fred_data scheduled__2023-04-09T00:00:00+00:00 [queued]>
[2023-11-02T10:07:13.501-0500] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: economics_data_pipeline.pull_and_save_fred_data scheduled__2023-04-09T00:00:00+00:00 [queued]>
[2023-11-02T10:07:13.501-0500] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-11-02T10:07:13.509-0500] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): pull_and_save_fred_data> on 2023-04-09 00:00:00+00:00
[2023-11-02T10:07:13.519-0500] {standard_task_runner.py:57} INFO - Started process 97260 to run task
[2023-11-02T10:07:13.527-0500] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'economics_data_pipeline', 'pull_and_save_fred_data', 'scheduled__2023-04-09T00:00:00+00:00', '--job-id', '33', '--raw', '--subdir', 'DAGS_FOLDER/economics_data_pipe.py', '--cfg-path', '/var/folders/3l/sjtljk5j23ng0k_w12x41kh80000gn/T/tmpwuv9annt']
[2023-11-02T10:07:13.528-0500] {standard_task_runner.py:85} INFO - Job 33: Subtask pull_and_save_fred_data
[2023-11-02T10:07:13.561-0500] {task_command.py:416} INFO - Running <TaskInstance: economics_data_pipeline.pull_and_save_fred_data scheduled__2023-04-09T00:00:00+00:00 [running]> on host macbook-pro-207.local
[2023-11-02T10:07:13.599-0500] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='hermesadmin' AIRFLOW_CTX_DAG_ID='economics_data_pipeline' AIRFLOW_CTX_TASK_ID='pull_and_save_fred_data' AIRFLOW_CTX_EXECUTION_DATE='2023-04-09T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-04-09T00:00:00+00:00'
[2023-11-02T10:07:13.601-0500] {fred_utils.py:34} INFO - Fetching data from URL: https://api.stlouisfed.org/fred/series/observations?series_id=GDP&api_key=5c78bd237c041fcf6bebdae4f8e05905&file_type=json
[2023-11-02T10:11:29.968-0500] {local_task_job_runner.py:294} WARNING - State of this instance has been externally set to failed. Terminating instance.
[2023-11-02T10:11:29.973-0500] {process_utils.py:131} INFO - Sending Signals.SIGTERM to group 97260. PIDs of all processes in the group: [97260]
[2023-11-02T10:11:29.973-0500] {process_utils.py:86} INFO - Sending the signal Signals.SIGTERM to group 97260
[2023-11-02T10:12:29.982-0500] {process_utils.py:149} WARNING - process psutil.Process(pid=97260, name='Python', status='running', started='10:07:13') did not respond to SIGTERM. Trying SIGKILL
[2023-11-02T10:12:29.987-0500] {process_utils.py:86} INFO - Sending the signal Signals.SIGKILL to group 97260
[2023-11-02T10:12:29.995-0500] {process_utils.py:79} INFO - Process psutil.Process(pid=97260, name='Python', status='terminated', exitcode=<Negsignal.SIGKILL: -9>, started='10:07:13') (97260) terminated with exit code Negsignal.SIGKILL
[2023-11-02T10:12:29.996-0500] {standard_task_runner.py:172} ERROR - Job 33 was killed before it finished (likely due to running out of memory)
